client/control/sass/control.scss

```diff
@@ -260,3 +269,12 @@
     font-weight: 800;
   }
 }
+
+/* Buttons */
+.button-icon {
+  background: none;
+  border: none;
+  cursor: pointer;
+  padding: 0;
+  transform: translate(0, -1px);
```

Comment:
The 1px transform is entirely visual. I thought it looked a little bit better raised up.

The `-moz-` prefix hasn't been needed [since Firefox 16](https://developer.mozilla.org/en-US/docs/Web/CSS/transform#Browser_compatibility). I think we can safely leave it off.

Plus, the preferred way for us to deal with prefixes is something like autoprefixer, which would just deal with all this.